app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -194,6 +227,73 @@ private fun TokenPropertyHeader(tokenProperty: TokenProperty<*>, modifier: Modif
     }
 }
 
+@Composable
+private fun CodeColumn(codeExample: String, modifier: Modifier = Modifier) {
+    val context = LocalContext.current
+    val clipboardManager = LocalClipboardManager.current
+    var isExpanded by rememberSaveable { mutableStateOf(false) }
+    val linkStateDescription = stringResource(if (isExpanded) R.string.app_common_expanded_a11y else R.string.app_common_collapsed_a11y)
+    val linkContentDescription = stringResource(R.string.app_tokens_viewCodeExample_label)
+
+    Column(modifier = modifier) {
+        Box(modifier = Modifier
+            .fillMaxWidth()
+            .clearAndSetSemantics {
+                contentDescription = linkContentDescription
+                stateDescription = linkStateDescription
+                onClick {
+                    isExpanded = !isExpanded
+                    true
+                }
+            }) {
+            Row(
+                modifier = Modifier.padding(horizontal = OudsSpaceKeyToken.Fixed.Medium.value, vertical = OudsSpaceKeyToken.Fixed.Short.value),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.spacedBy(OudsSpaceKeyToken.PaddingInline.WithArrow.Short.value)
+            ) {
+                Text(
+                    text = stringResource(R.string.app_tokens_viewCodeExample_label),
+                    style = OudsTypographyKeyToken.LabelStrongLarge.value
+                )
+                Icon(
+                    modifier = Modifier.rotate(if (isExpanded) 180f else 0f),
+                    painter = painterResource(R.drawable.ic_chevron_down),
+                    tint = OudsColorKeyToken.Brand.Primary.Default.value,
+                    contentDescription = null
+                )
+            }
+        }
+        AnimatedVisibility(visible = isExpanded) {
+            Box(
+                modifier = Modifier
+                    .padding(horizontal = OudsSpaceKeyToken.Fixed.Medium.value)
```

Comment:
The horizontal padding of the `Box` and the `Row` can be factorized at the `Column` level.